pxr/usd/usdGeom/mesh.h

```diff
@@ -54,28 +54,69 @@ class SdfAssetPath;
 
 /// \class UsdGeomMesh
 ///
-/// Encodes a mesh surface whose definition and feature-set
-/// will converge with that of OpenSubdiv, http://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html.
-/// Certain interpolation ("tag") parameters are not yet supported.
+/// Encodes a mesh with optional subdivision properties and features.
 /// 
+/// Meshes are conceptually defined in terms of vertices, edges and faces.
+/// As a point-based primitive, the terms "vertex" and "point" are synonymous,
+/// as are associated terms such as "point index" and "vertex index", etc.
+/// The encoding defines the mesh topology completely in terms of a set of
+/// faces and their defining face-vertices (see _faceVertexCounts_ and
+/// _faceVertexIndices_).  No boundary representation or additional
+/// connectivity between vertices, edges and faces is required or
+/// constructed, so no adjacency or neighborhood queries are available.
+///
 /// A key property of this mesh schema is that it encodes both subdivision
-/// surfaces, and non-subdivided "polygonal meshes", by varying the
-/// \em subdivisionScheme attribute.
+/// surfaces and simpler polygonal meshes. This is achieved by varying the
+/// _subdivisionScheme_ attribute, which is set to specify Catmull-Clark
+/// subdivision by default, so polygonal meshes must always be expliclty
+/// declared. The available subdivision schemes and additional subdivision
+/// features encoded in optional attributes conform to the feature set of
+/// OpenSubdiv
+/// (https://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html).
 /// 
-/// \section UsdGeom_Mesh_Normals A Note About Normals
+/// \anchor UsdGeom_Mesh_Primvars
+/// <b>A Note About Primvars</b>
+///
+/// The following lists clarifies the number of elements for and the
+/// interpolation behavior of the different primvar interpolation types
+/// for meshes:
+///
+/// \li <b>constant</b> One element for the entire mesh; no interpolation.
+/// \li <b>uniform</b> One element for each face of the mesh; elements are
+///     typically not interpolated but are inherited by other faces derived
+///     from a given face (via subdivision, tessellation, etc.).
+/// \li <b>varying</b> One element for each vertex or point of the mesh;
```

Comment:
This is one place where the distinction between point and vertex pays off:  this interpolation mode is specified per-point, _not_ per-vertex... the RenderMan (adopted by us) `vertex` interpolation mode is unfortunate because it is also per-point.

Reply:
Noted, thanks -- "vertex" now removed.